core/src/main/java/dk/alexandra/fresco/framework/builder/binary/BinaryUtilityBuilder.java

```diff
@@ -0,0 +1,41 @@
+package dk.alexandra.fresco.framework.builder.binary;
+
+import dk.alexandra.fresco.framework.Computation;
+import dk.alexandra.fresco.framework.value.SBool;
+import java.io.PrintStream;
+import java.util.List;
+
+public interface BinaryUtilityBuilder {
```

Comment:
See similar comment about numeric utility builder - this interface ought not to exist